src/main/kotlin/device/application/DeviceSearcher.kt

```diff
@@ -12,6 +12,6 @@ class DeviceSearcher(
      * @see Device
      * @return null or a device instance if it exists
      */
-    fun get(identifier: String) = get(Identifier(identifier))
-    private fun get(identifier: Identifier) = repository.search(identifier)
+    fun search(identifier: String) = search(Identifier(identifier))
```

Comment:
I won't close this pull request but, for the next ones, be aware that here you are doing a refactor (that isn't a must-to-have in this use case*) + adding a feature. This breaks the statements discussed on the project rules ( #43  ).

The next time, if you can, do two different pull requests (one for the refactor and another for the feature). 

* I didn't mean that this refactor was unnecessary, I mean that, in the context of adding the "remove device" feature wasn't necessary and could be implemented on another pull request.

Reply:
You're right, I'll do it that way from now on.